lib/common/components/FeedLabel.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import React from 'react'
+import tinycolor from 'tinycolor2'
+import Icon from '@conveyal/woonerf/components/icon'
+import { connect } from 'react-redux'
+
+import { deleteLabel } from '../../manager/actions/labels'
+import type { Label } from '../../types'
+import type {ManagerUserState} from '../../types/reducers'
+import ConfirmModal from '../../common/components/ConfirmModal'
+import LabelEditorModal from '../../manager/components/LabelEditorModal'
+
+export type Props = {
+  checked?: boolean,
+  deleteLabel: Function,
+  label: Label,
+  onClick?: Function,
+  small: boolean,
+  user?: ManagerUserState
+}
+export type State = {}
+
+// Generate lightened/darkened versions of a color for use in text
+// and border rendering
+const getComplementaryColor = (cssHex, strength) => {
+  const color = tinycolor(cssHex)
+
+  const complementary = color.isDark()
+    ? color.lighten(strength)
+    : color.darken(strength + 10)
+  return complementary.toHexString()
+}
+
+class FeedLabel extends React.Component<Props, State> {
+  _onConfirmDelete = () => {
+    this.props.deleteLabel && this.props.deleteLabel(this.props.label)
+  }
+
+  _onClickDelete = () => {
+    this.refs.deleteModal.open()
+  }
+
+  _onClickEdit = () => {
+    this.refs.editModal.open()
+  }
+
+  render () {
+    const {small, label, checked, onClick: labelOnClick, user} = this.props
+    const projectAdmin =
+      user &&
+      user.permissions &&
+      user.permissions.isProjectAdmin(label.projectId)
+
+    return (
+      <div
```

Comment:
I tried this, and while it would probably help, the Bootstrap `label` stylings interfere too strongly with the custom styling, and I don't want to complicate the CSS any more.

Reply:
We can do it anyway though if you think it is a good idea

Reply:
I'm talking about the plain HTML `<label>` that goes with `<input>` controls.

Reply:
I think there is global bootstrap css which causes even those to be styled. It doesn't look to be more than padding and font weight, but I am weary of adding even more styling. Do you think it's worth it?